storages/python/python-storage-plugins/yaml.py

```diff
@@ -148,19 +148,28 @@ def from_bytes(cls, buffer, id=None):
         )
 
     @classmethod
-    def to_bytes(cls, inst, soft7=True, with_uuid=False):
+    def to_bytes(cls, inst, options=None):
         """Save instance `inst` to bytes (or bytearray) object.  Optional.
 
         Arguments:
             inst: Instance to save.
-            soft7: Whether to structure metadata as SOFT7.
-            with_uuid: Whether to include UUID in the output.
+            options: Supported options:
+            - `soft7`: Whether to structure metadata as SOFT7.
+            - `with_uuid`: Whether to include UUID in the output.
```

Comment:
`soft7` and `with_uuid` are not the same thing. With `soft7=True` dimensions and properties will be dicts, otherwise they will be arrays. `with_uuid=True` simply adds an `uuid` key to the return representation.

Reply:
For me the option descriptions look different:

        Arguments:
            location: Path to YAML file.
            options: Supported options:
            - `mode`: Mode for opening.  Valid values are:
                - `a`: Append to existing file or create new file (default).
                - `r`: Open existing file for read-only.
                - `w`: Truncate existing file or create new file.
            - `soft7`: Whether to save using SOFT7 format.
            - `single`: Whether the input is assumed to be in single-entity form.
              If "auto" (default) the form will be inferred automatically.
            - `with_uuid`: Whether to include UUID when saving.

But it would be possible to also use the `single` for output.